dts/bindings/spi/intel,intel-spi.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2018 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Intel SPI Controller
+version: 0.1
+
+description: >
+     This is a representation of the Intel spi node
+
+inherits:
+    !include spi.yaml
+
+properties:
+    compatible:
+      constraint: "intel,intel-spi"
```

Comment:
Is there anything more detailed or descriptive than 'intel,intel-spi'?  Seems a bit too generic

Reply:
Ok, maybe we should but a bit more description  in 'description:' field about what SoC this applies to right now.